### DISCUSSION TITLE
Using spkrec-xvect-voxceleb For Verification (All Comparisons Scores Are High)

### DISCUSSION THREAD

**Discussion by MattyB95** [speechbrain/speechbrain#2155] [Q&A]
I sought to perform speaker verification tasks using the default spkrec-xvect-voxceleb and spkrec-ecapa-voxceleb models. The ECAPA model works as expected. However, the xvect scores high no matter the comparison.

For example:

```
from speechbrain.pretrained import SpeakerRecognition

verification = SpeakerRecognition.from_hparams(source="speechbrain/spkrec-xvect-voxceleb",
                                               savedir="pretrained_models/spkrec-xvect-voxceleb")
score, prediction = verification.verify_files('id10001\\00001.wav', 'id10002\\00003.wav')  # Different Speakers
print(score, prediction)
score, prediction = verification.verify_files('id10001\\00001.wav', 'id10001\\00002.wav')  # Same Speaker
print(score, prediction)
```

Output: 
tensor([0.9933]) tensor([True])
tensor([0.9982]) tensor([True])

I would expect the different speakers to score much lower than the same speaker, but regardless of which audio files I compare, all scores are > 0.99. I need to do something entirely differently when using xvect for verification.

**Comment by maxpain** [comment]
The same problem
